src/draw/DefaultDrawer.js

```diff
@@ -494,6 +495,46 @@ class DefaultDrawer {
     node.select('.component-container>rect').attr('data-parentId', ({ data }) => data.id);
   }
 
+  /**
+   * Initialize component height and width and store them in it's drawOptions.
```

Comment:
Initialize component height and width then store them in its drawOptions.

Reply:
done